charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,193 @@
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  secrets: []
+  # List of volumes that can be mounted by containers belonging to the pod.
+  additionalVolumes: []
+  # Pod volumes to mount into the container's filesystem.
+  additionalVolumesMounts: []
+
+  # Remove the above empty additionalVolumes and additionalVolumesMounts, and then set additionalVolumes and additionalVolumesMounts as below if you want to collect node exporter metrics
+# additionalVolumes:
+#   - name: hostProc
+#     hostPath:
+#       path: /proc/
+#   - name: hostSys
+#     hostPath:
+#       path: /sys/
+# additionalVolumesMounts:
+#   - mountPath: /host/sys
+#     mountPropagation: HostToContainer
+#     name: hostSys
+#     readOnly: true
+#   - mountPath: /host/proc
+#     mountPropagation: HostToContainer
+#     name: hostProc
+#     readOnly: true
+
+  # Set a limit of memory that Tail plugin can use when appending data to the Engine.
+  # If the limit is reach, it will be paused; when the data is flushed it resumes.
+  # You can find more details here: https://docs.fluentbit.io/manual/pipeline/inputs/tail#config
+  # if the inbound traffic is less than 2.4Mbps, setting memBufLimit to 5MB is enough
+  # if the inbound traffic is less than 4.0Mbps, setting memBufLimit to 10MB is enough
+  # if the inbound traffic is less than 13.64Mbps, setting memBufLimit to 50MB is enough
+  input:
+    tail:
+      memBufLimit: 5MB
+    # uncomment below nodeExporterMetrics section if you want to collect node exporter metrics
+    nodeExporterMetrics:
+#     tag: node_metrics
+#     scrapeInterval: 15s
+#     path:
+#       procfs: /host/proc
+#       sysfs: /host/sys
+
+  # Configure the output plugin parameter in FluentBit.
+  # You can set enable to true to output logs to the specified location.
+  output:
+#  You can find more supported output plugins here: https://github.com/fluent/fluent-operator/tree/master/docs/plugins/fluentbit/clusteroutput
+    es:
+      enable: false
+#     host: "<Elasticsearch url like elasticsearch-logging-data.kubesphere-logging-system.svc>"
+#     port: 9200
+#     logstashPrefix: ks-logstash-log
+#     path: ""
+#     bufferSize: "4KB"
+#     index: "fluent-bit"
+#     httpUser:
+#     httpPassword:
+#     logstashFormat: true
+#     replaceDots: false
+#     enableTLS: false
+#     tls:
+#       verify: On
+#       debug: 1
+#       caFile: "<Absolute path to CA certificate file>"
+#       caPath: "<Absolute path to scan for certificate files>"
+#       crtFile: "<Absolute path to private Key file>"
+#       keyFile: "<Absolute path to private Key file>"
+#       keyPassword:
+#       vhost: "<Hostname to be used for TLS SNI extension>"
+    kafka:
+      enable: false
+#     brokers: "<kafka broker list like xxx.xxx.xxx.xxx:9092,yyy.yyy.yyy.yyy:9092>"
+#     topics: ks-log
+    stdout:
+      enable: true
+
+  # Configure the default filters in FluentBit.It's our customized lua contained log format converter.
```

Comment:
Some typos, I would not use "our", just say it's a lua filter to handle containerd logs.

Any reason not to make a parser btw?

Reply:
The purpose of adding `containerd` here is to make the data output format consistent with `docker`. Other imperfections will be addressed in future PRs.

Reply:
Could you review it again?